web/portal/platform/src/entities/SpecialNumber/ForeignKeyGetter.tsx

```diff
@@ -0,0 +1,22 @@
+import { ForeignKeyGetterType } from '@irontec/ivoz-ui/entities/EntityInterface';
+import { autoSelectOptions } from '@irontec/ivoz-ui/entities/DefaultEntityBehavior';
+import { SpecialNumberPropertyList } from './SpecialNumberProperties';
+
+/** TODO remove this file unless you need to change default behaviour **/
```

Comment:
remove this file

Reply:
it is needed for the form

Reply:
use the default one 
`import { foreignKeyGetter } from '@irontec/ivoz-ui/entities/DefaultEntityBehavior';`